src/addon/chrome/chromeFiles/locale/en_US/locale.dtd

```diff
@@ -150,13 +150,13 @@
 <!ENTITY options.proxy.description
 "Instead of  a direct connection you may use a SOCKS Proxy to transparently and securely traverse a firewall.">
 <!ENTITY options.proxy.description.nameresolution
-"With SOCKSv4 hostnames are always resolved on the local machine. SOCKSv5 also supports resolving hostnames by the SOCKS proxy. If you can trust the SOCKSv5 proxy, the latter is considered to be more secure and should be preferred.">
+"With SOCKSv4 hostnames are always resolved on the local machine. SOCKS v5 also supports resolving hostnames by the SOCKS proxy. If you can trust the SOCKSv5 proxy, the latter is considered to be more secure and should be preferred.">
```

Comment:
It is interesting how many variants are existing. Wikipedia uses SOCK5 (https://en.wikipedia.org/wiki/SOCKS). In the official RFCs they use SOCKS V5 (e.g. https://tools.ietf.org/html/rfc1929) with a capital. Interestingly enough SOCKS v5 (with a small v) seems to be also commonly used.

Reply:
What would you like to do?

Reply:
To save you typing what you want to do here, and then us updating it, and then you accepting it; feel free to change it how you see fit. Thank you